README.md

```diff
@@ -81,7 +81,9 @@ This will create a message like this:
 
 ![example block message](https://git.io/fjDWR)
 
-You can also check out the [`slackerduty`](https://github.com/CGA1123/slackerduty) project for some example,
+See [`./examples`](./examples/) for more worked examples.
```

Comment:
Planning to add a bit more guidance here with regard to updating the `config.rb`, etc.